k8s/src/main/scala/io/buoyant/k8s/PortMapLogging.scala

```diff
@@ -0,0 +1,35 @@
+package io.buoyant.k8s
+
+import com.twitter.logging.Logger
+
+private[k8s] trait PortMapLogging {
+  def nsName: String
+  def serviceName: String
+  def logPortDiff[A,B](oldPorts: Map[A, B], newPorts: Map[A, B]): Unit =
+    if (log.isLoggable(Logger.TRACE)) {
+      for {
+        port <- oldPorts.keySet
+        oldValue <- oldPorts.get(port)
+      } newPorts.get(port) match {
+        case Some(`oldValue`) =>
```

Comment:
any reason for the backtick escaped name?

Reply:
yeah, this is matching the case where the value contained in the `Some` is _equal to_ the value of `oldValue`, not shadowing the binding of `oldValue` to a new value.

Reply:
(how many times can I say "value" in one sentence?)

Reply:
Oh!  I definitely missed that.  Would `case Some(value) if value == oldValue =>` be more clear perhaps?